Code/Tools/AssetProcessor/native/tests/platformconfiguration/platformconfigurationtests.cpp

```diff
@@ -664,3 +665,24 @@ TEST_F(PlatformConfigurationUnitTests, PlatformConfigFile_IsPresent_Found)
     ASSERT_TRUE(config.AddPlatformConfigFilePaths(platformConfigList));
     ASSERT_EQ(platformConfigList.size(), 1);
 }
+
+TEST_F(PlatformConfigurationUnitTests, Test_MetaFileTypes_AssetImporterExtensions)
```

Comment:
Casing was a problem with the config system when built. It's probably worth adding a test here to make sure casing is handled correctly. The bug before was, a file named someFile.FBX wasn't being handled by the scene builder using the config for the extension because it was only looking for lowercase fbx.

Reply:
so you want the test to add some more entries like 'aAa' and/or 'bBB' right?

Reply:
got it, i will check whether we already have an existing test that checks file casing issue because if we do then i do not see value adding another test and just ensuring that the metadata info is populated correctly should be enough.

Reply:
Yeah, I think a test should be added to check for it, too. There's a chance that internal logic needs to be updated, or this format of test wouldn't be the right place to catch it if the casing correction happens elsewhere in the chain. The existing code assumes all extensions will be lowercase in the m_supportedFileTypeExtensions list, and before checking if a file matches, it to-lowers that file's extension in SceneImportRequestHandler::LoadAsset